plugins/kubernetes/app/models/kubernetes/client_config_file.rb

```diff
@@ -107,6 +111,10 @@ def client
         Kubeclient::Client.new(cluster.url, api_version, ssl_options: ssl_options)
       end
 
+      def ext_client
+        Kubeclient::Client.new(cluster.server + '/apis', 'extensions/v1beta1', ssl_options: ssl_options)
+      end
```

Comment:
FYI, it wasn't immediately clear to me what "ext" was short for.  I guessed "External", before I saw that it involved extensions.